spec/lcm/slow/m_to_n_spec.rb

```diff
@@ -0,0 +1,89 @@
+require_relative 'support/connection_helper'
+require_relative 'support/configuration_helper'
+require_relative 'support/s3_helper'
+
+require_relative 'support/project_helper'
+require_relative 'shared_examples_for_synchronization_bricks'
+
+require 'active_support'
+require 'active_support/core_ext'
+
+describe 'Contains M to N relation' do
```

Comment:
Can we test this also at the unit-test level?

Reply:
it is now covered with unit tests too